phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/common/widgets/lists/ListWidgetEntry.java

```diff
@@ -28,6 +29,20 @@
 public class ListWidgetEntry<E> {
     private static final Logger LOGGER = LoggerFactory.getLogger(ListWidgetEntry.class);
 
+    private static URI DEFAULT_MINIATURE;
+    private static URI WINE_MINIATURE;
+    private static URI CONTAINER_MINIATURE;
+
+    static {
```

Comment:
I would like to find a way to initialize the values without a static block. Do you have an idea?